packages/ui-extensions/src/surfaces/checkout/components/Image/Image.doc.ts

```diff
@@ -22,7 +22,7 @@ const data: ReferenceEntityTemplateSchema = {
       tabs: [
         {
           code: './examples/basic-image.example.html',
-          language: '',
```

Comment:
thank you for finding this one, brought it over [here](https://github.com/shop/world/pull/157885/files#diff-040b307a3ad4c8f7e579df2daa0011a91879664895e68c250932a4959cb086cfL25)